ui/settingsContainer.ts

```diff
@@ -217,15 +217,14 @@ export class Settings {
                 for (let i = 0; i < notifications.length; i++) {
                     let notification = notifications[i]
                     let enabled = this.manager.enabledNotifications.includes(`${value}??${notification.id}`)
-                    if (notification.forced) enabled = true;
                     let checkbox = createBoldCheckbox(notification.id + " - ", notification.description, enabled, async (b) => {
                         utils.removeItem(this.manager.enabledNotifications, `${value}??${notification.id}`)
                         if (b) {
                             this.manager.enabledNotifications.push(`${value}??${notification.id}`)
                         }
                         let enabledKey = `${window.location.host}_notificationsEnabled`
                         await GM.setValue(enabledKey, JSON.stringify(this.manager.enabledNotifications))
-                    }, notification.forced)
```

Comment:
i still think the checkboxes for topics that are on top level should be disabled, because it's going to re-enable them every time you load the page anyways

Reply:
Yeah, but until the page is reloaded, the user can at least unsubscribe from some of the topics. And maybe they will be smart enough to actually add the template to always load and not get re-subscribed to all the topics

Reply:
i still think its bad UX for it to let you unsubscribe if it just resubscribes anyways — users should be forced to add the template to always load if they want to unsubscribe from topics

Reply:
I guess, but the only weird thing about it is in order to get rid of the template in the address bar after you add it to always load is to reload the entire page. Then again, it's probably better to reload to unsubscribe than to reload and get re-subscribed to everything.

Reply:
> I guess, but the only weird thing about it is in order to get rid of the template in the address bar after you add it to always load is to reload the entire page. Then again, it's probably better to reload to unsubscribe than to reload and get re-subscribed to everything.

yeah, better to do this than have reloading=resubscribed to everything